build.ps1

```diff
@@ -109,7 +109,12 @@ function Invoke-Build([string]$Path, [switch]$Clean, [string]$Command, [switch]$
         "build" {
             Install-Rustup $toolchain
             Install-RustToolchain $toolchain
-            rustup component add --toolchain $Toolchain rustfmt
```

Comment:
Why the removal of the installation of `rustfmt`?

Reply:
we do install when running `fmt` I think it was accidentally placed here in `build`